README.md

```diff
@@ -7,15 +7,36 @@ Local Time makes it easy to display times and dates to users in their local time
 1. Add `gem 'local_time'` to your Gemfile.
 2. Include `local-time.js` in your application's JavaScript bundle.
 
-    Using the asset pipeline:
-    ```js
-    //= require local-time
-    ```
-    Using the [local-time npm package](https://www.npmjs.com/package/local-time):
-    ```js
-    import LocalTime from "local-time"
-    LocalTime.start()
-    ```
+   1. Using the asset pipeline:
+      ```js
+      //= require local-time
+      ```
+
+   1. Using the [local-time npm package](https://www.npmjs.com/package/local-time):
+      ```js
+      import LocalTime from "local-time"
+      LocalTime.start()
+      ```
+
+   1. Using Ruby on Rails 6 with Webpacker:
+
+      1. Add `local-time` to `package.json`
+         ```json
+         {
+           "dependencies": {
+             "local-time": "^2.1.0"
+           }
+         }
+         ```
+      1. Run `yarn install` to install npm packages
+         ```sh
+         $ yarn install
+         [...] Resolving packages ...
+         ```
```

Comment:
The two above can be combined into

```
yarn add local-time
```

Reply:
Thank you @adammiribyan ! I am new to yarn and did not yet know about yarn install :-) I updated the PR.